daliuge-engine/dlg/drop.py

```diff
@@ -2052,13 +2057,13 @@ class PlasmaDROP(AbstractDROP):
     """
 
     object_id = dlg_string_param("object_id", None)
-    plasma_path = dlg_string_param("plasma_path", "/tmp/plasma")
+    plasma_path = dlg_string_param("plasma_path", "/tmp/.dlg/workspace/plasma")
 
     def initialize(self, **kwargs):
         object_id = self.uid
         if len(self.uid) != 20:
             object_id = np.random.bytes(20)
-        if self.object_id is None:
+        if not self.object_id:
```

Comment:
I noticed an issue with plasma where an empty string id didn't trigger using the object id. Truthy logic will treat empty string and None as False.